cmd/template/cluster/provider/templates/aws/cluster.go

```diff
@@ -54,7 +54,7 @@ func NewClusterCRs(config ClusterCRsConfig) (ClusterCRs, error) {
 	// the workload cluster name may be provided by the user.
 	{
 		if config.ClusterName == "" {
-			generatedName, err := key.GenerateName(config.EnableLongNames)
+			generatedName, err := key.GenerateName(true)
```

Comment:
This generates a longer version of the resource name.
Can it be like this or is it better to generate 5-character names?